AnnoDesigner/Undo/AddRemoveOperation.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using AnnoDesigner.Core.DataStructures;
+using AnnoDesigner.Models;
+
+namespace AnnoDesigner.Undo
```

Comment:
My opinion is to move all possible operations to a common namepsace like `AnnoDesigner.Undo.Operations` or somehting like that.